### PR TITLE
addCohort

### PR DESCRIPTION
**Pull request by lubnaabd**
relates #9

### CODE REVIEW

**Review by ali-7 — APPROVED** [review]

File: public/css/cohorts.css [hunk 1]
```diff
@@ -116,6 +119,7 @@ input[type="text"] {
   display: inline-block;
   border: 1px solid #ccc;
   border-radius: 4px;
+  width: 60%;
   box-sizing: border-box;
   margin-bottom: 2%;
```
Comment (NouraldinS) [inline public/css/cohorts.css]: Don't use relative margins and paddings.

File: public/js/cohort.js [hunk 2]
```diff
@@ -1,6 +1,85 @@
...
   addCoh.classList.toggle('sectionAddcoh--visible');
-}
+};
+
+const displayErr = (errElem, errMsg) => {
+  errElem.textContent = errMsg;
+};
+
+const checkNam = () => {
+  if (names.value === '') {
```
Comment (NouraldinS) [inline public/js/cohort.js]: same as:
```js
if (name.value) {
```

File: public/js/cohort.js [hunk 3]
```diff
@@ -1,6 +1,85 @@
...
+  }
+};
+
+names.addEventListener('focusout', checkNam);
+descriptions.addEventListener('focusout', checkdes);
+githublinks.addEventListener('focusout', checkgit);
+imgURls.addEventListener('focusout', checkimg);
+
+addCohorts.addEventListener('click', (e) => {
+  if (checkNam() && checkdes() && checkgit() && checkimg()) {
```
Comment (NouraldinS) [inline public/js/cohort.js]: What are `checkNam`, `checkdes` and `checkgit`?

File: src/views/cohort.hbs [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <section class="content">
+ <button class="button button2" onclick="addCohort()">ADD Cohort</button>
   <section>
     {{#if err}}
     <h3>{{err}}</h3>
```
Comment (NouraldinS) [inline src/views/cohort.hbs]: Fix indentation.

File: src/views/cohort.hbs [hunk 5]
```diff
@@ -36,17 +37,32 @@
...
-      <label for="/img"> /img</label>
-      <input type="text" id="/img" name="/img" placeholder=" Link /img">
-      <input type="submit" value="ADD Cohort">
-    </div>
-  </section>
\ No newline at end of file
+      <section class="sectionAddcoh" id="addCoh">
+        <h2>Add Cohort</h2>
+        <div class="cohDiv">
+
```
Comment (NouraldinS) [inline src/views/cohort.hbs]: Remove unnecessary newline.

File: src/views/cohort.hbs [hunk 6]
```diff
@@ -36,17 +37,32 @@
       </table>
```
Comment (NouraldinS) [inline src/views/cohort.hbs]: Fix file's indentation.

**Review by NouraldinS — CHANGES_REQUESTED** [review]

**Review by HemaSAli — APPROVED** [review]

**Review by NouraldinS — APPROVED** [review]

**Pull request merged**